environment.yaml

```diff
@@ -21,5 +21,6 @@ dependencies:
 - bottleneck=1.3.2
 - zarr=2.8.3
 - pip:
+  - pooch 
   - git+https://github.com/dgergel/xsd@458f292dab6e8a6584659e97a66c37e028da2b7a
-  - xclim 
+  - git+https://github.com/ClimateImpactLab/xclim@add_analog_downscaling
```

Comment:
If you can, would you be able to swap out the `@add_analog_downscaling` and replace it with a full commit hash, like `@e087dd21b88fc5594471beb0f00ad932bfb529af`?

Reply:
Do you think we could keep it pointing to the branch for now, just in case there are a few small updates? I may update the handling of edge cases in xclim...

Reply:
We talked about this last week so marking as resolved

Reply:
Would you please, note what commit you're on now for this package? Just in the PR?